.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,37 @@
 <!--- Please review your changes in preview mode -->
 <!--- Provide a general summary of your changes in the Title above -->
 
-#### Issue-ID
+#### Issue-ID (If there is an associated issue)
 <!--- Replace XXXX below (2 places) with the actual JIRA id -->
 * **[PP-XXXX](https://pbspro.atlassian.net/browse/PP-XXXX)**
 
-#### Problem description
-* Describe the issue being fixed
+#### Problem description 
+* Clearly describe the problem as it affects the customer
+* Clearly describe the steps to reproduce the problem
+* If the issue is intermittent and not easily reproducible, describe the situation/scenarios/configurations where it was likely triggered
+* Specify the exact platform where the bug is observed, for example CentOS 7.2
```

Comment:
I think this bullet is not needed as we have a dedicated section for "Affected Platform(s) and PBS Version